hazelcast/src/hazelcast/client/compact.cpp

```diff
@@ -1995,6 +1995,66 @@ default_schema_service::send_schema_response_decode(
                                           boost::hash<boost::uuids::uuid>>>();
 }
 
+bool
+default_schema_service::has_any_schemas() const
+{
+    return replicateds_.size();
+}
+
+std::ostream&
+operator<<(std::ostream& os, const std::vector<schema>& schemas)
+{
+    os << "Map {";
```

Comment:
```suggestion
    os << "Schemas {";
```

Reply:
Current implementation matches with Java. This operator overloads `<<` only for `Map`. Do you still want me to change ?